example/cumulus-tf/ecs_hello_world_workflow.tf

```diff
@@ -12,7 +12,7 @@ module "hello_world_service" {
 
   cluster_arn                           = module.cumulus.ecs_cluster_arn
   desired_count                         = 1
-  image                                 = "cumuluss/cumulus-ecs-task:1.4.0"
+  image                                 = "jlkovarik/cumulus-ecs-task:1.5.1"
```

Comment:
Are we not intending to publish this to `cumuluss`?

Reply:
Yes, just as soon as that PR is complete.

Reply:
03e55b6d7